reframe/chunker.go

```diff
@@ -35,7 +34,6 @@ func newChunker(chunkSizeFunc func() int, nonceGenFunc func() []byte) *chunker {
 	}
 	ch := &chunker{
 		chunkByContextId: make(map[string]*cidsChunk),
-		chunkByCid:       make(map[cid.Cid]*cidsChunk),
```

Comment:
This index isn't required anymore as references to chunks get recorded in `cidNode`. Will allow to save some RAM on not needing to store the key twice.